src/styles/block/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  & :not(:last-child) {
+    margin-right: 4px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), there should be no space before the colon. Please remove the space in `& :not(:last-child)` to correct it to `&:not(:last-child)`.

Reply:
it makes sense to write it as suggested: `&:not(:last-child)` and move it inside `&__star` - this style related to the star element